### PR TITLE
switching back to pip-20.1.1 to fix error in building Python-2.7.18-G…

### PR DESCRIPTION
**Pull request by algiar**
…CCcore-10.2.0

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -43,9 +43,9 @@ exts_list = [
         'use_pip': False,
         'checksums': ['c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b'],
     }),
-    ('pip', '20.2.3', {
+    ('pip', '20.1.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb:46]: @algiar Please use `20.3.4` instead (the latest version of `pip` which still supports Python 2), see discussion in https://github.com/easybuilders/easybuild-easyconfigs/issues/12016#issuecomment-787052638.

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by Mormacill** [comment]
Test report by @Mormacill
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
host02.mebus - Linux oracle linux server 8.3, x86_64, Intel(R) Core(TM) i3-8100 CPU @ 3.60GHz, Python 3.6.8
See https://gist.github.com/7b771a41503bd9f066c488041d5f549b for a full test report.

**Comment by Mormacill** [comment]
Test report by @Mormacill
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node01.mebus - Linux centos linux 7.9.2009, x86_64, Intel(R) Core(TM) i3-8100 CPU @ 3.60GHz, Python 3.6.8
See https://gist.github.com/78a6ab54e2362d9f5c277cf82d9ccd04 for a full test report.

**Comment by Mormacill** [comment]
Test report by @Mormacill
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node03.mebus - Linux ubuntu 18.04, x86_64, Intel(R) Core(TM) i3-8100 CPU @ 3.60GHz, Python 3.6.9
See https://gist.github.com/64d9b4cd256f183ccfd9fb5e9a7ccae1 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node313.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e096b60d6f702f87714b7eb12e11575d for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @algiar !

**Pull request merged**